clap_derive/tests/default_value.rs

```diff
@@ -17,3 +17,39 @@ fn auto_default_value() {
     let help = get_long_help::<Opt>();
     assert!(help.contains("[default: 0]"));
 }
+
+#[test]
+fn flag_default_value_does_not_accept_empty() {
```

Comment:
Please write this test in `clap` crate without the derive stuff.

Reply:
Sure thing, as a heavy structopt user looking forward to clap 3, I found this one easier to write to express my desired result. Will move it!